src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -98,14 +98,14 @@
     /** Compiled regexp to match Javadoc tags that take an argument. */
     private static final Pattern MATCH_JAVADOC_ARG = CommonUtil.createPattern(
             "^\\s*(?>\\*|\\/\\*\\*)?\\s*@(throws|exception|param)\\s+(\\S+)\\s+\\S*");
-
-    /** Compiled regexp to match first part of multilineJavadoc tags. */
-    private static final Pattern MATCH_JAVADOC_ARG_MULTILINE_START = CommonUtil.createPattern(
-            "^\\s*(?>\\*|\\/\\*\\*)?\\s*@(throws|exception|param)\\s+(\\S+)\\s*$");
+    /** Compiled regexp to match Javadoc tags with argument but with missing description. */
+    private static final Pattern MATCH_JAVADOC_ARG_MISSING_DESCRIPTION =
+        CommonUtil.createPattern("^\\s*(?>\\*|\\/\\*\\*)?\\s*@(throws|exception|param)\\s+"
+            + "(\\S[^*]*)(?:(\\s+|\\*\\/))?");
```

Comment:
`(\\S+)\\s*$` is converted to `(\\S[^*]*)(?:(\\s+|\\*\\/))?`.

Reply:
(\\S+)\\s*$ expression did not cover this expression case:
 /** @param x*/
so left unchanged